apps/authentication-service/src/resources/clients/clients.test.ts

```diff
@@ -0,0 +1,36 @@
+import { Client } from "auth0";
+import {
+  getClient,
+  createClient,
+  deleteClient,
+} from "@/resources/clients/clientsService";
+
+describe("Clients test", () => {
+  const clientId: string = "ucp-test-client";
+  const clientName = "Ucp Test Client";
+  const clientDesc = "For unit testing";
+
+  it("Test Client Creation", async () => {
+    const client: Client | Error = await createClient({
+      name: clientName,
+      description: clientDesc,
+    });
+
+    expect(client).not.toBe(undefined);
+    expect((client as Client).client_id).toBe(clientId);
```

Comment:
I think it would be more valuable to ensure that the whole object returned from all these tests shows up like it should. Just testing that it has a matching clientId doesn't give me much confidence

Reply:
How can I test that the entire object was returned? What are ways that Jest provides to test for that?

Reply:
@wesrisenmay-mx ^

Reply:
expect(client).toEqual(expectedObject)

I usually like to create test data files and export their values to msw handlers as well as tests. This makes it easier to import them in my unit tests and make sure the correct object is being returned.

Reply:
Okay, thanks. I think I've seen this done in other repos/projects. I'll dig around and find an example.